### PR TITLE
Files Added: Assignment 1,2,3,4

### PR DESCRIPTION
**Pull request by Richa199**

### CODE REVIEW

**Review by mubeen2710 — APPROVED** [review]

**Review by vdongre304 — APPROVED** [review]

**Review by yogesh545 — APPROVED** [review]

**Review by Jestinxavier — APPROVED** [review]

**Review by Jestinxavier — APPROVED** [review]

**Review by nikita0699 — APPROVED** [review]

File: Richa/HTML/Assignment1/ASS1.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    </head>
+    <body>
+        <h1>HTML</h1>
+        <h2><b>Introduction</b></h2>
+        <p>
+            HTML is a standard markup language for Web Pages.
+        </p>  
+    </body>
+</html>
+
```
Comment (JafarPathan) [inline Richa/HTML/Assignment1/ASS1.html:14]: remove extra line

File: Richa/HTML/Assignment3/RichaA3_1.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                <ul>
+                    <li>Mihika</li>
+                    <li>Pallavi</li>
+                    <li>Reva</li>
+                </ul>
+                <li>MI-MEAN-9</li>
+                <ul>
+                    <li>Anil</li>
+
+                    <li>Manoj</li>
```
Comment (JafarPathan) [inline Richa/HTML/Assignment3/RichaA3_1.html:32]: remove extra lines

File: Richa/HTML/Assignment4/RichaA4_3.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+            <tr>
+                <td rowspan="3">Tuesday</td>
+                <td class="b">8:00am</td>
+                <td class="b">11:00am</td>
+                <td rowspan="3"><div><div class="a">XPath</div><div class="a">XSL Transformations</div></div></td>
+            </tr>
+            <tr>
+                <td class="b">11:00am</td>
+                <td class="d">2:00pm</td>
+
```
Comment (JafarPathan) [inline Richa/HTML/Assignment4/RichaA4_3.html:70]: remove extra lines

File: Richa/HTML/Assignment4/RichaA4_3.html [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+            </tr>
+            <tr>
+                <td class="b">11:00am</td>
+                <td class="d">2:00pm</td>
+                
+            </tr>
+            <tr>
+                <td class="d">2:00pm</td>
+                <td class="c">5:00pm</td>
+
```
Comment (JafarPathan) [inline Richa/HTML/Assignment4/RichaA4_3.html:75]: remove extra lines

**Review by JafarPathan — COMMENTED** [review]
check all files remove all extra lines and spaces
do proper indentation in all files 
always add single line at end of file

**Pull request merged**